shared/reducers/tracker.js

```diff
@@ -457,6 +435,14 @@ function remoteProofToProof (rp: RemoteProof, lcr: ?LinkCheckResult): Proof {
     statusMeta = proofStatusToSimpleProofMeta(lcr.proofResult.status)
   }
 
+  const isTracked = !!(lcr && lcr.diff && lcr.diff.type === identify.TrackDiffType.none && !lcr.breaksTracking)
+
+  // whatevz, tracked a broken thing
+  if (lcr && lcr.proofResult && lcr.proofResult.status !== identify.ProofStatus.ok && isTracked) {
+    diffMeta = metaIgnored
+    statusMeta = null
```

Comment:
ah I misread that `!==` as a `===`. Disregard the below comment about changes to proofStatusToSimpleProofMeta. I think the changes to `trackDiffToSimpleProofMeta` still hold though